metafacture-xml/src/main/java/org/metafacture/xml/SimpleXmlEncoder.java

```diff
@@ -351,6 +372,16 @@ protected static void writeEscaped(final StringBuilder builder, final String str
         builder.append(XmlUtil.escape(str, false));
     }
 
+    private boolean isDefaultNamespace(final String ns) {
+        return ns.isEmpty() || ns.equals(DEFAULT);
+    }
+
+    private void propertiesToMap(final Properties properties) {
+        for (final Entry<Object, Object> entry : properties.entrySet()) {
+            namespaces.put(entry.getKey().toString(), entry.getValue().toString());
+        }
```

Comment:
`Properties` is a `Map`, so this could also be simplified (optional):

```suggestion
        properties.forEach((k, v) -> namespaces.put(k.toString(), v.toString()));
```